soql-analyzer/src/main/scala/com/socrata/soql/analyzer2/rewrite/RemoveTrivialSelects.scala

```diff
@@ -19,6 +19,9 @@ class RemoveTrivialSelects[MT <: MetaTypes] private () extends StatementUniverse
           var exprs = OrderedMap[AutoColumnLabel, Column]()
 
           for((label, namedExpr) <- selectList) {
+            if(namedExpr.hint != None) { // if a selected column has a hint, it's not trivial
```

Comment:
Hm. This feels like it'll make for very few cases of trivial selects. Not sure how I feel about that.

Reply:
Hm.